spiff-arena-common/src/spiff_arena_common/coverage.py

```
@@ -17,6 +58,7 @@ def cov_tasks(test_cases):
                     yield coverage_spec_id, task["task_spec"]
         for _, sp in state["subprocesses"].items():
             id = sp["spec"]
```

Ruff (0.15.12)

[error] Variable `id` is shadowing a Python builtin

(A001)